platform/src/main/java/org/hillview/table/NoOpStringConverter.java

```diff
@@ -0,0 +1,31 @@
+package org.hillview.table;
+
+import org.hillview.table.api.IStringConverter;
+import org.hillview.table.api.IStringConverterDescription;
+
+import javax.annotation.Nullable;
+
+/**
+ * A string converter that does nothing
+ */
+public class NoOpStringConverter implements IStringConverter, IStringConverterDescription {
+    private static final NoOpStringConverter INSTANCE = new NoOpStringConverter();
```

Comment:
Why not make this public and remove getInstance()?

Reply:
While not strictly necessary here, I did it out of habit (it's cleaner to access such singletons behind a method in the event we want to say, log every access to the API).